Bigtable/src/BigtableClient.php

```diff
@@ -0,0 +1,206 @@
+     *           Console.
+     *     @type float $requestTimeout Seconds to wait before timing out the
+     *           request. **Defaults to** `0` with REST and `60` with gRPC.
+     *     @type int $retries Number of retries for a failed request.
+     *           **Defaults to** `3`.
+     * }
+     * @throws GoogleException If the gRPC extension is not enabled.
+     */
+    public function __construct(array $config = [])
+    {
+        $this->requireGrpc();
+        $config += [
+            'scopes' => [
+                self::FULL_CONTROL_SCOPE,
+                self::ADMIN_SCOPE
+            ],
+            'projectIdRequired' => true
+        ];
+        $this->connection = new Grpc($this->configureAuthentication($config));
+        $this->setLroProperties(new LongRunningConnection($this->connection), [
+            [
+                'typeUrl' => 'type.googleapis.com/google.bigtable.admin.instance.v2.UpdateInstanceMetadata',
+                'callable' => function ($instance) {
+                    $name = InstanceAdminClient::parseName($instance['name'])['instance'];
+                    return $this->instance($name, $instance);
+                }
+            ], [
+                'typeUrl' => 'type.googleapis.com/google.bigtable.admin.instance.v2.CreateInstanceMetadata',
+                'callable' => function ($instance) {
+                    $name = InstanceAdminClient::parseName($instance['name'])['instance'];
+                    return $this->instance($name, $instance);
+                }
+            ]
+        ]);
+    }
+
+    /**
+     * Lazily instantiate an instance.
+     *
+     * Example:
+     * ```
+     * $instance = $bigtable->instance('my-instance');
+     * ```
+     *
+     * @param string $instanceId The instance ID
+     * @param array $instance [optional] {
+     *     Configuration options
+     *
+     *     @type string $displayName **Defaults to** the value of $instanceId.
+     *     @type int $instanceType Possible values are represented by the following constants:
+     *           `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_PRODUCTION` and
+     *           `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_DEVELOPMENT`.
+     *           **Defaults to** using `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_UNSPECIFIED`.
+     *     @type array $labels as key/value pair ['foo' => 'bar']. For more information, see
+     *           [Using labels to organize Google Cloud Platform resources](https://cloudplatform.googleblog.com/2015/10/using-labels-to-organize-Google-Cloud-Platform-resources.html).
+     * }
+     * @return Instance
+     */
+    public function instance($instanceId, array $instance = [])
+    {
+        return new Instance(
+            $this->connection,
+            $this->lroConnection,
+            $this->lroCallables,
+            $this->projectId,
+            $instanceId,
+            $instance
+        );
+    }
+
+    /**
+     * Cluster metadata.
```

Comment:
Description is not pretty useful.
I'd recommend saying a bit more. Does it return a metadata for Cluster or else?